chipsec/cfg/pch_3xx.xml

```diff
@@ -60,8 +60,8 @@ XML configuration file for the 300 series PCH
     <!-- ~~~~~~~~~~~~~~~~~~~~~~~~~~~~ -->
 
     <!-- Sideband Register Access Registers -->
-    <register name="SBREG_BAR" type="pcicfg" bus="0" dev="0x1F" fun="1" offset="0x10" size="4" desc="Sideband Register Access BAR">
-      <field name="RBA" bit="24" size="8" desc="Register Base Address"/>
+    <register name="SBREG_BAR" type="pcicfg" bus="0" dev="0x1F" fun="1" offset="0x10" size="8" desc="Sideband Register Access BAR">

```

Comment:
I'd add a comment here indicating that this also includes SBREG_BARH, and that this should be done for 1xx.xml and 2xx.xml as well.

Reply:
@martingalvan would you like to make the PR for the change in 1xxx and 2xx?

Reply:
Sure. I won't be able to do this immediately though, but I can definitely contribute this.